spec/system/admin/reports/pay_your_suppliers_spec.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+require "system_helper"
+
+RSpec.describe "Pay Your Suppliers Report" do
+  include ReportsHelper
+
+  let(:hub) { create(:distributor_enterprise) }
+  let(:order_cycle) { create(:open_order_cycle, distributors: [hub]) }
+  let(:product) { order.products.first }
+  let(:variant) { product.variants.first }
+  let(:supplier) { variant.supplier }
+  let(:current_user) { hub.owner }
+  let!(:order) do
+    create(:completed_order_with_totals, distributor: hub, order_cycle:, line_items_count: 1)
+  end
+  let(:params) { { display_summary_row: true } }
+  let(:report) { Reporting::Reports::Suppliers::Base.new(current_user, { q: params }) }
+  let(:report_table_rows) { report.rows }
```

Comment:
These are not needed.

Reply:
😬

Reply:
It's fixed here:
https://github.com/openfoodfoundation/openfoodnetwork/pull/12879/commits/03c8e2cee3ff6dee58185b6f87e498b4c7397c49#diff-36e7c39c383994d08652e0192d25c4f629d776cc106cb7cec3268bbb8cbd59bfL17-L19